python/cuml/cluster/hdbscan/hdbscan.pyx

```diff
@@ -1013,7 +1125,7 @@ class HDBSCAN(Base, ClusterMixin, CMajorInputTagMixin):
             "prediction_data"
         ]
 
-    def get_attributes_names(self):
+    def get_attr_names(self):
         attr_names = ['labels_', 'probabilities_', 'cluster_persistence_',
                       'condensed_tree_', 'single_linkage_tree_',
                       'outlier_scores_']
```

Comment:
Should be good based on what I read in the hdbscan library documentation, but always good to double check nothing was forgotten here.

Reply:
I doubled checked as well